src/components/02-molecules/RoomTimeline/index.js

```diff
@@ -1,13 +1,8 @@
 import React, { PropTypes } from 'react';
-import { connect } from 'react-redux';
-
 import styles from './styles.scss';
-
 import Meeting from '../Meeting';
 
-import { timelineMeetingRequested } from '../../../actions';
-
-const RoomTimeline = ({ room, onRoomTimelineClick }) => {
+const RoomTimeline = ({ room }) => {
```

Comment:
Need to agree: are we uginng classes for components or not. 50/50 is not an answer